config/crd/resources.yaml

```diff
@@ -140,6 +140,31 @@ spec:
                 required:
                 - name
                 type: object
+              diskEncryption:
+                description: DiskEncryption is the configuration to Enable/disable
+                  disk encryption for cluster nodes.
+                properties:
+                  enableOn:
+                    default: none
+                    description: Enable/disable disk encryption on master nodes, worker
+                      nodes, or all nodes.
+                    enum:
+                    - none
+                    - all
+                    - masters
+                    - workers
+                    type: string
+                  mode:
+                    description: The disk encryption mode to use.
+                    enum:
+                    - tpmv2
+                    - tang
+                    type: string
+                  tangServers:
+                    description: JSON-formatted string containing additional information
+                      regarding tang's configuration
+                    type: string
+                type: object
```

Comment:
This is more of a generic question.
Why do we have 2 different CRDs? Don't we have a way to define it once and use it twice?